app/src/main/res/values/strings.xml

```diff
@@ -365,6 +366,12 @@
   <string name="bookmark_empty">You haven\'t added any bookmarks</string>
   <string name="provider_bookmarks_location">Bookmarks</string>
   <string name="log_collection_started">Log collection started. Please RESTART the app, perform action that you wish to log, and then tap \'Send log file\' again</string>
-  <string name="no_uploads">Welcome to Commons!\n
-Upload your first media by touching the camera or gallery icon above.</string>
+  <string name="no_uploads">Welcome to Commons!\n Upload your first media by touching the camera or gallery icon above.</string>
+
+  <string name="deletion_reason_1">I uploaded it by mistake</string>
```

Comment:
rename string `name`. You can probably use 

`deletion_reason_uploaded_by_mistake`
`deletion_reason_publicly_visible`
`deletion_reason_bad_for_my_privacy`
`deletion_reason_no_longer_want_public`
`deletion_reason_not_interesting`